.travis.yml

```diff
@@ -7,13 +7,20 @@ cache:
   yarn: true
   directories:
   - node_modules
+addons:
+  sonarcloud:
+    organization: "dannylekim"
```

Comment:
pretty sure the organization is Billsnap 

![image](https://user-images.githubusercontent.com/26320256/75834721-de059800-5d8a-11ea-8c8f-b3ed2fc3b0f3.png)

Reply:
Odd. Sonarcloud gave me this to copy paste.

Reply:
yeah it used to be dannylekim but I changed it, so I guess it should be billsnap now?